src/api/app/controllers/webui/obs_factory/staging_projects_controller.rb

```diff
@@ -0,0 +1,71 @@
+module Webui::ObsFactory
+  class StagingProjectsController < ApplicationController
+    respond_to :json, :html
+
+    before_action :require_distribution
+    before_action :require_project_name, only: [:show]
+
+    def index
+      respond_to do |format|
+        format.html do
+          @staging_projects = ::ObsFactory::StagingProjectPresenter.sort(@distribution.staging_projects_all)
+          @backlog_requests = ::ObsFactory::Request.with_open_reviews_for(by_group: @distribution.staging_manager, target_project: @distribution.name)
+          @requests_state_new = ::ObsFactory::Request.in_state_new(by_group: @distribution.staging_manager, target_project: @distribution.name)
+
+          staging_project = Project.find_by_name("#{@distribution.project}:Staging")
+          dashboard_package = Package.find_by_project_and_name(staging_project.name, 'dashboard')
+
+          if dashboard_package
+            if dashboard_package.file_exists?('ignored_requests')
+              file = ::Backend::Api::Sources::Package.file(staging_project.name, dashboard_package.name, 'ignored_requests')
+              @ignored_requests = YAML.load(file)
+            end
+          else
+            flash[:notice] = "package dashboard doesn't exist for #{staging_project}"
```

Comment:
A dashboard package is not a requirement and as such this will be annoying for projects without one.

Reply:
Okey, I will remove it